packages/gravity/blade-runner/src/plan/env/redis.test.ts

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { expect } from 'chai';
+import { Redis, type RedisOptions } from 'ioredis';
+
+import { type TaggedType } from '@dxos/codec-protobuf';
+import { PublicKey } from '@dxos/keys';
+import { type TYPES } from '@dxos/protocols';
+import { RpcPeer } from '@dxos/rpc';
+import { afterTest, describe, openAndClose, test } from '@dxos/test';
+
+import { REDIS_PORT, createRedisRpcPort } from './util';
+
+/**
+ * NOTE(mykola): This test is disabled because it requires a running Redis server.
+ *               `redis-server --port 6378` to start a Redis server on port 6378.
+ * TODO(mykola): Mock Redis server.
+ */
+describe.skip('Redis', () => {
+  test('two redis client can exchange messages', async () => {
+    const client = await setupRedisClient();
+    const server = await setupRedisClient();
+
+    const key = 'message-' + PublicKey.random().toHex();
+    const value = 'hello';
+    const message = server.blpop(key, 0);
+
+    await client.rpush(key, value);
+
+    expect(await message).to.deep.equal([key, value]);
+  });
+
+  test('RPC connection between two redis clients', async () => {
+    const [alicePort, bobPort] = await createLinkedRedisPorts();
+
+    const alice = new RpcPeer({
```

Comment:
nit: semantic names might be better? user1, user2.
unless the name is helpful to reason about a specific complex interaction
(i.e., ok to use alice/bob in specialized crypto test flows.